### PR TITLE
Feature/SiteBanner

### PR DESCRIPTION
**Pull request by RamMahe1110**

### CODE REVIEW

File: src/components/SiteBanner.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      setTimeout(() => {
+        this.changeSiteBannerVisibility(false);
+      }, 200);
+    }
+  },
+  created() {
+    this.fetchSiteBannerData();
+  },
+  watch: {
+    getBannerData: function(siteBannerData) {
```
Comment (atharva3010) [inline src/components/SiteBanner.vue:52]: Why is this watcher needed? getters are reactive, you can trigger these operations in the getter itself.

**Comment by paulchrisluke** [comment]
This banner needs to match our specifications as indicated in the issue: https://github.com/whynotearth/foodouken.com/issues/94

image


Right now, the font, color, size all don't match. We should be using global styles for colors and fonts @atharva3010 . I am very concerned about our fonts and colors, as we seem to run into inconsistancy stuff here. Inconistant sizing/spacing on colors/fonts is one of the easiest ways to compromise a brand image. Is there a way we can get more global with this?

**Comment by atharva3010** [comment]
@paulchrisluke We are using global styling, we never used this site banner colour, so it's not in our app's colour scheme. @RamMahe1110 Can you add the colour from figma as a new key in our tailwind.config.js? Regarding sizing, please use `text-5xl` for the font, and instead of having `p-4`, use `px-4`.

The end result should look something like this:
image

File: src/components/SiteBanner.vue [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  },
+  methods: {
+    ...mapActions('home', [
+      'fetchSiteBannerData',
+      'changeSiteBannerVisibility'
+    ]),
+    onSiteBannerCloseClick() {
+      const opacityColln = [0.9, 0.7, 0.5, 0.3, 0.1];
+      for (let i = 0; i < opacityColln.length; i++) {
+        document.querySelector('.site-banner').style.opacity = opacityColln[i];
```
[the remark this answers is not in the document]
Comment (kbventures) [reply src/components/SiteBanner.vue:41]: Hi @atharva3010, could you kindly clarify what refs are. I haven't used before.

File: src/store/modules/home.js [hunk 3]
```diff
@@ -41,6 +49,22 @@ const actions = {
         }
       );
     });
+  },
+  fetchSiteBannerData({ commit }) {
```
[the remark this answers is not in the document]
Comment (kbventures) [reply src/store/modules/home.js:53]: Should I go ahead and remove these two lines?

File: src/components/SiteBanner.vue (hunk 1 above)
Comment (kbventures) [reply src/components/SiteBanner.vue:52]: @atharva3010 I haven't been trained in vuex. Can you advise on how I can get this moving forward?

**Comment by paulchrisluke** [comment]
banner completed

**Pull request closed without merging**